library/common-library/templates/_staging.tpl

```diff
@@ -0,0 +1,26 @@
+{{- /*
+Abstraction of the nrStaging toggle.
+This helper allows to override the global `.global.nrStaging` with the value of `.nrStaging`.
+Returns "true" if `nrStaging` is enabled, otherwise "" (empty string)
+*/ -}}
+{{- define "common.nrStaging" -}}
+{{- /* `get` will return "" (empty string) if value is not found, and the value otherwise, so we can type-assert with kindIs */ -}}
+{{- if (get .Values "nrStaging" | kindIs "bool") -}}
+    {{- if .Values.nrStaging -}}
+        {{- /*
+            We want only to return when this is true, returning `false` here will template "false" (string) when doing
+            an `(include "common.nrStaging" .)`, which is not an "empty string" so it is `true` if it is used
+            as an evaluation somewhere else.
+        */ -}}
+        {{- .Values.nrStaging -}}
+    {{- end -}}
+{{- else -}}
+{{- /* This allows us to use `$global` as an empty dict directly in case `Values.global` does not exists */ -}}
+{{- $global := index .Values "global" | default dict -}}
+{{- if get $global "nrStaging" | kindIs "bool" -}}
+    {{- if $global.nrStaging -}}
+        {{- $global.nrStaging -}}
+    {{- end -}}
+{{- end -}}
+{{- end -}}
+{{- end -}}
```

Comment:
There is a "library/common-library/values.yaml" committed, do we need it for the CI, linting, etc?
If so I would keep it, but I would add a comment saying why it is empty and why we need it

Reply:
can we remote this or add a comment? just to avoid having a completely empty file "flying" 
![Screen Shot 2022-03-02 at 11 03 39 AM](https://user-images.githubusercontent.com/43335750/156340132-34e2d191-88d3-4f90-a08f-18103a886470.png)

Reply:
It is needed for `helm lint` to work.